js/common.js

```diff
@@ -892,6 +892,11 @@ var templateSelection = function (selection) {
 var templateItilStatus = function(option) {
```

Comment:
First argument passed to callback of function referenced in `templateSelection` is `false` when element do not match selection. Not sure if it is a bug in select2 or not.
```suggestion
var templateItilStatus = function(option) {
    if (option === false) {
        // option is false when element does not match searched terms
        return null;
    }
```